ch1/Makefile

```
@@ -6,5 +6,6 @@ server:
 	docker compose up -d
 
 server-clean:
-	docker compose down -v
-	docker rmi ch1-data-generator
+	echo "------------Cleaning Chapter 1 resources------------"
+	docker compose down -v || true
+	docker rmi ch1-data-generator || true
```

```suggestion
	docker rmi -f ch1-data-generator
```
`docker compose down -v` 은 어떻게 바꿀 수 있을지 모르겠지만 이미지 제거는 이렇게 할 수 있을 것 같습니다!

---

아 이건 이미지 없을 때 나는 에러를 무시하기 위함이였습니다